### PR TITLE
fixed compose file issue for backend

### PR DESCRIPTION
**Pull request by YoBulkMaster**

### CODE REVIEW

**Review by ciminelli — APPROVED** [review]

**Comment by YoBulkMaster** [comment]
@ciminelli  I have updated the read me with curl file download. The main compose file is updated with image pull approach. There is a build image docker-compose file in case someone wants to build it in local machine. The front end docker-compose file is added with build for backend as, there might be some backend changes which is needed the front end to run.

Kindly have a look.. thanks

File: README.md [hunk 1]
```diff
@@ -71,8 +71,9 @@ sudo usermod -a -G docker $USER && \
 sudo curl -L "https://github.com/docker/compose/releases/download/v2.19.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && \
 sudo mv /usr/local/bin/docker-compose /usr/bin/docker-compose && \
 sudo chmod +x /usr/bin/docker-compose && \
-git clone https://github.com/tableflowhq/tableflow.git && cd tableflow && \
-cp .env.example .env && \
+mkdir tableflow && cd tableflow && \
+curl --ssl-no-revoke -O https://github.com/tableflowhq/tableflow/blob/main/docker-compose.base.yml -O https://github.com/tableflowhq/tableflow/blob/main/docker-compose.yml -O https://github.com/tableflowhq/tableflow/blob/main/.env.example
```
Comment (ciminelli) [inline README.md:75]: Thoughts on curl vs wget? I might prefer wget for simplicity assuming its as widely supported.

Comment (ciminelli) [reply README.md:75]: `wget https://raw.githubusercontent.com/tableflowhq/tableflow/main/{.env.example,docker-compose.yml,docker-compose.base.yml}`

**Review by ciminelli — COMMENTED** [review]
Looks great, just one suggestion on `curl` vs `wget`. Next can you document the process on how to push tags and trigger a release? Not sure if we'll just do this manually or we can have a small script to increment the version, or somehow do this in github.

File: README.md (hunk 1 above)
Comment (YoBulkMaster) [reply README.md:75]: Yes, I thought of using wget. Then I saw in line 71, we have used curl. So, to keep it uniform, thought of using curl. Please suggest

Comment (ciminelli) [reply README.md:75]: @YoBulkMaster I would move both to wget if you can!

Comment (YoBulkMaster) [reply README.md:75]: Perfect! Let me do that then

Comment (YoBulkMaster) [reply README.md]: I have moved the curl into wget. For the tagging, once we push the tag, the latest version will automatically be set to this tag in docker hub. So, whenever user pulls the images with docker-compose they will have the latest image running in their system.

Comment (YoBulkMaster) [reply README.md]: @ciminelli please have a look and we can have a quick call for our policies on the tagging

**Review by ciminelli — APPROVED** [review]

**Pull request merged**